packages/video_player/video_player/lib/video_player.dart

```diff
@@ -244,8 +261,30 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   @visibleForTesting
   int get textureId => _textureId;
 
+  /// Set the cache size in bytes. Default is maxSize of `100 * 1024 * 1024`
+  /// and maxFileSize of `10 * 1024 * 1024`.
+  ///
+  /// Cache used only for network data source.
+  ///
+  /// Throws StateError if you try to set the cache size twice. You can only set it once.
+  static void setCacheSize(int maxSize, int maxFileSize) {
```

Comment:
This method is a little confusing to me. Why don't we just make this part of `intialize` below, with optional named parameters, or alternatively make it part of the `.network` ctor?

Reply:
We should also document what happens if this value changes from initialization to initialization.

Is the cache per controller or per application?

If the max size changes, do we trim the cache immediately or when we try to add more to it? Or do we not trim it at all?

How can we evict a file from the cache? How can we evict all files from the cache?